### ISSUE TITLE
VS Code handling of Blade language doesn't support Emmet properly

### ISSUE THREAD

**Issue by mushmelty** [microsoft/vscode#122123]
1. Create a new file with a `.blade.php` extension.
2. Confirm that "Blade" is selected as the Language Mode.
3. Enter any of the standard Emmet shortcut commands (such as `p` for `<p></p>` or `h` for `<h1></h1>`, `<h2></h2>`, etc
4. Emmet snippets will not appear, regardless of settings (except for the `!` snippet to generate HTML boilerplate).

Expected behavior:
Emmet snippets would appear, as consistent with user settings.

Notes:
I have noticed that the Emmet snippets WILL appear if I enter a space after the entered text, and then immediately backspace to the text. Therefore, if I were to type "span", and then a space, and then backspace... the snippets would appear as expected.

If I switch the Language Mode to PHP, then Emmet snippets will appear as expected. It's only through VS Code's handling of the Blade language mode that such snippets don't appear properly.

**Comment by rzhao271** [comment]
@mushmelty which extensions are you using for Blade support? I noticed there are a few in the Marketplace.
With https://marketplace.visualstudio.com/items?itemName=onecentlin.laravel-blade, I am unable to reproduce the issue in VS Code Insiders. Does the issue still occur for you using the latest version of VS Code?

**Comment by mushmelty** [comment]
I am using that Laravel Blade snippet extension "Laravel Blade Snippets" (Winnie Lin), and am using Code version 1.55.2, and the behavior is the same as I had initially described. Specifically, Emmet snippets don't appear unless I manually hit the space bar, then backspace once... which is unusual behavior that I haven't seen elsewhere.

For example, if I wish to create `<tr class="border-b"></tr>` using Emmet, I would have to type `tr.border-b` then space, then backspace... and then the expected Emmet snippets appear below (of which I could then select by hitting enter as usual).

The (Winnie Lin) Blade snippets appear initially as expected, but Emmet snippets only appear after spacing then backspacing.

**Comment by rzhao271** [comment]
Can you take a screencap showing the problem, e.g. with gifcap https://gifcap.dev/?
I'm still unable to reproduce it with a sample from https://codingspoint.com/foreach-loop-laravel-8-blade-file-example/:

![recording (4)](https://user-images.githubusercontent.com/7199958/117171162-7d536000-ad7f-11eb-96f3-1ca1061fb3d1.gif)

**Comment by mushmelty** [comment]
Here is an animation that shows what is going on with my machine. You'll notice that the Emmet snippets aren't displaying until I tap the space bar and then backspace. A very few of the Emmet snippets will work without this procedure; specifically the "!" for HTML boilerplate, and the "h1" – "h6" tags for headers — but not until I actually type a number, which is not normal behavior, and leads me to think that there is some problem with alphabet characters (and not numbers or symbols) not being detected. Just something I've noticed.

Here's the animation I captured that demonstrates the behavior:

![github](https://user-images.githubusercontent.com/37992344/117232600-73ebf580-ae5c-11eb-8dea-d1a263a34161.gif)

**Comment by rzhao271** [comment]
Hmm I'm still unable to reproduce the issue :c
Can you re-report this issue through `Help > Issue Reporter`? 
That way, I can also see all the extensions you have.
Alternatively, can you run extension bisect? https://code.visualstudio.com/blogs/2021/02/16/extension-bisect#:~:text=The%20Extension%20bisect%20utility%20in%20VS%20Code%20was,which%20commit%20in%20the%20repository%20introduced%20an%20issue.

**Comment by mushmelty** [comment]
After I begin Extension Bisect (and all extensions are disabled), VS Code treats a file as `Plain Text`, and when I go to manually switch it to Blade, there is no option for Blade files. However, I just treated this as the "good" option and continued. Eventually the process determined Livewire Language Support to be the offender.

The Issue Reporter won't work with this particular extension, and when I went to the Issues section of the GitHub repository, there are already several open issues regarding to the same situation. After disabling / uninstalling, VS Code is once again working as expected.
